src/fontra/client/web-components/designspace-location.js

```diff
@@ -111,13 +111,24 @@ export class DesignspaceLocation extends UnlitElement {
         elements.push(html.hr());
         continue;
       }
+      const values = axis.values ? axis.values : [];
```

Comment:
Could it work like this, too?
```suggestion
      const values = axis.values;
```
If so, consider not useing an additional variable, but just use `axis.values` directly.

Thit would be my preference.

Reply:
Lets keep it for now, we don't have `values` yet. An empty array is needed for creating range slider.

Reply:
Creating range slider with values `undefined` doesn't work. It must be either an array or not to be existing in initial properties.

Reply:
Can you make it to work? I think `values = undefined` or `values = null` should be a valid thing for the range slider.

Reply:
Now that it works, please get rid of the `values` variable, and use `axis.values` throughout. (There's one place where you already do that accidentally.)

Reply:
I deleted' em